### PR TITLE
[WIP] Make Killercoda exercise the LTS Quick Start

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

* Give the "Quick Start" title to the Killercoda End-to-End exercise
* Replace installation instructions in that section with links to the appropriate documentation.  The instructions that are here are not adequate and having duplicate installation instructions throughout the doc set creates maintenance headaches
* Modify the "Installation" section to make it friendly for people installing a private Keptn instance for study and demos, as well as for people installing a Keptn production instance.

Note that this PR does not update the exercise to use Keptn 1.0.x-LTS.  That will be done in a separate PR.

https://github.com/keptn/keptn.github.io/issues/1464

### CODE REVIEW

File: content/docs/1.0.x/install/k8s/index.md [hunk 1]
```diff
@@ -6,20 +6,30 @@ weight: 10
...
-This includes major commercial Kubernetes providers
-as well as k3s and k3d, which allow you to install a small Kubernetes cluster
-on your laptop for study and demonstration purposes.
 
+* This includes major commercial Kubernetes providers
+  that are appropriate for production instances of Keptn.
+* k3s, k3d, and Minikube allow you to install a small Kubernetes cluster
+  on your laptop for study and demonstration purposes.
+
+*Hint* Be sure to check [Kubernetes support & Cluster size](../k8s-support)
```
Comment (afzal442) [inline content/docs/1.0.x/install/k8s/index.md:25]:
```suggestion
**Note:** Make sure to check [Kubernetes support & Cluster size](../k8s-support)
```

File: content/docs/1.0.x/install/k8s/index.md [hunk 2]
```diff
@@ -6,20 +6,30 @@ weight: 10
...
+  to ensure that Keptn is compatible with the Kubernetes version you are running
+  and that your Kubernetes cluster has enough resources for Keptn.
+  In particular, ensure that you include adequat storage capacity
+  for the Persistent Volume Claims (PVCs).
+  We recommend at least 20GB to 30GB of storage capacity for any Keptn installlation.
+  Larger installations running more complex projects may need additional storage capacity.
+  See [Check resources](../troubleshoting/#check-resources) for more information.
```
Comment (afzal442) [inline content/docs/1.0.x/install/k8s/index.md:32]:
```suggestion
  See [Check resources](../troubleshooting/#check-resources) for more information.
```

**Review by afzal442 — CHANGES_REQUESTED** [review]

File: content/docs/1.0.x/install/k8s/index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/k8s/index.md:25]: done

File: content/docs/1.0.x/install/k8s/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/k8s/index.md:32]: Done (blushing) ;-)

**Review by agardnerIT — APPROVED** [review]

**Pull request merged**